_projects/autosegct.md

```diff
@@ -0,0 +1,90 @@
+---
+layout: project_page
+title: Radiotherapy Auto Segmentation
+status: Evaluating
+image: assets/img/projects/autosegct_logo.png
+summary: Single application for all in-house auto-segmentation contours
+csc-lead: <a href="/team_member/Anil.html">Anil Mistry</a> <a href="/team_member/Tom.html">Tom Roberts</a>
+modality: Radiotherapy planning CT 
+pathology: Cancer
+rationale: Contouring organs on CT's scans for radiotherapy treatments is vital for patient safety, however is very time consuming for Oncologists. AI models have been developed locally to automate the CT image segmentation. 
+patient-pathway: 
+training-data: Gold standard dual-labelled CT's and structure sets of patients treated historically at Guy's Cancer Centre. 
+errors:  N/A
+goals:  Provide an application that can clinically deploy all locally developed nnUNEt segmentation models successfully into the radiotherapy treatment planning pathway. 
+success-criteria: Reduction in time spent contour by Radiation oncologists. Improves consistency in contouring OAR's. Increased accuracy of OAR contours.  
+---
+
+<b>Clinical lead</b>: Christopher Thomas, Teresa Guerrero Urbano
+<br>
+<br> **Rationale:**
+
+AI auto-contouring tools have improved dramatically in the last few years, and all major treatment planning system vendors are investing in this technology along with independent start-ups and NHS Trusts [NICE, 2023].
+
+GSTT are in a strong position to develop our own auto-contouring pipeline using the AI expertise within RT physics, clinical oncology, and the Clinical Scientific Computing (CSC) team, strong links with King’s College London (KCL) academics and outputs from three KCL PhD students creating AI auto-contouring models.
+
+Segmentation models have been created as output of the extensive research activity generated from Guys Cancer centre. This application will integrate the following models for evaluation in routine clinical care.
```

Comment:
```suggestion```
Segmentation models have been created as output of the extensive research activity generated from Guy's Cancer Centre. This application will integrate the following models for evaluation in routine clinical care.
```suggestion```

Reply:
Resolved